dotBloch/Assets/Classes/Constants.cs

```diff
@@ -0,0 +1,28 @@
+﻿class Constants
+{
+    public static class symbol{
+
+        public const string bloch_sphere = "|Ψ> = ";
+        public const string quantum_one = " |1>";
+        public const string quantum_zero = " |0> ";      
+    }
+
+    public static class error{

```

Comment:
Zastanawiam się, czy nie była by lepsza jakaś osobna klasa wyjątków

Reply:
It's only a message for Debug.LogArguemnt.Exception/Debug.Log - not necessary to create another class.